### DISCUSSION TITLE
PageSerializer/Page<Object>

### DISCUSSION THREAD

**Discussion by pkokliaguine-opentext-com** [micronaut-projects/micronaut-data#2563] [Q&A]
Why nothing from the Slice (Page's parent) is included in serialization?

`"content": [...],
    "pageable": {
        "number": 5,
        "sort": {},
        "size": 10
    },
    "totalSize": 136,
    "totalPages": 14,
    "pageNumber": 5,
    "numberOfElements": 10,
    "empty": false,
    "size": 10,
    "offset": 50`
now we have only 
`"content": [...],
    "pageable": {
        "number": 5,
        "sort": {},
        "size": 10
    },
    "totalSize": 136,
`

**Comment by dstepanov** [comment]
Looks like a mistake.
Please create a PR to add what do you think is needed to https://github.com/micronaut-projects/micronaut-data/blob/master/data-model/src/main/java/io/micronaut/data/model/PageSerializer.java
